tutor/templates/k8s/deployments.yml

```diff
@@ -352,7 +352,7 @@ spec:
       containers:
         - name: mongodb
           image: {{ DOCKER_IMAGE_MONGODB }}
-          args: ["mongod", "--nojournal", "--storageEngine", "wiredTiger"]
```

Comment:
Is this a breaking change? What's the impact of removing that option?

Reply:
I am not sure if we can classify it as a breaking change. Starting with Mongo 6.1, the journal module is always enabled https://www.mongodb.com/docs/manual/reference/configuration-file-settings-command-line-options-mapping/#changelog and the nojournal param is no longer available.